api/src/main/java/org/pmiops/workbench/cdr/dao/CustomCBCriteriaDaoImpl.java

```diff
@@ -435,10 +433,11 @@ public List<DbCardCount> findDomainCountsByNameEndsWithAndStandardAndDomains(
   @Override
   public List<DbCardCount> findDomainCountsByTermAndNameEndsWithAndStandardAndDomains(
       String term, List<String> endsWithList, Boolean standard, List<String> domains) {
+    StringBuilder stringBuilder = new StringBuilder(term);
+    domains.forEach(d -> stringBuilder.append(" [" + d + "_rank1]"));
     Object[][] params = {
       {BIND_VAR_STANDARD, standard},
-      {BIND_VAR_TERM, term},
-      {VAR_IN_DOMAINS, domains}
+      {BIND_VAR_TERM, stringBuilder.toString()}
```

Comment:
All the changes in this class address item #2

Reply:
Very nice!